modules/backup-hosted-cluster-virt.adoc

```diff
@@ -0,0 +1,95 @@
+
+.Procedure
+
+. Create a Velero backup resource by creating a YAML file that is similar to the following example:
++
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  name: hc-clusters-hosted-backup
+  namespace: openshift-adp
+  labels:
+    velero.io/storage-location: default
+spec:
+  includedNamespaces: <1>
+  - clusters
+  - clusters-hosted
+  includedResources:
+  - sa
+  - role
+  - rolebinding
+  - deployment
+  - statefulset
+  - pv
+  - pvc
+  - bmh
+  - configmap
+  - infraenv
+  - priorityclasses
+  - pdb
+  - hostedcluster
+  - nodepool
+  - secrets
+  - hostedcontrolplane
+  - cluster
+  - datavolume
+  - service
+  - route
+  excludedResources: [ ]
+  labelSelector: <2>
+    matchExpressions:
+    - key: 'hypershift.openshift.io/is-kubevirt-rhcos'
+      operator: 'DoesNotExist'
+  storageLocation: default
+  preserveNodePorts: true
+  ttl: 4h0m0s
+  snapshotMoveData: true <3>
+  datamover: "velero" <4>
+  defaultVolumesToFsBackup: false <5>
+----
++
+<1> This field selects the namespaces from the objects that will be backed up. Include namespaces from both the hosted cluster and the hosted control plane. In this example, `clusters` is a namespace from the hosted cluster and `clusters-hosted` is a namespace from the hosted control plane. By default, the `HostedControlPlane` namespace is `clusters-<hosted_cluster_name>`.
+<2> The boot image of the VMs that are used as the hosted cluster nodes are stored in large PVCs. To filter those PVCs out of the backup to reduce backup time and storage size, add this label selector.
+<3> This field and the `datamover` field enable the CSI VolumeSnapshots to be automatically uploaded to remote cloud storage.
+<4> This field and the `snapshotMoveData` field enable the CSI VolumeSnapshots to be automatically uploaded to remote cloud storage.
+<5> Set this value to `false` to safely backup the PVCs that you want.
+
+. Apply the changes to the YAML file by entering the following command:
++
+[source,terminal]
+----
+$ oc apply -f <backup_file_name>.yaml
+----
++
+Replace `<backup_file_name>` with the name of your file.
+
+. Monitor the backup process in the backup object status and in the Velero logs.
++
+** To monitor the backup object status, enter the following command:
++
+[source,terminal]
+----
+$ watch "oc get backup -n openshift-adp <backup_file_name> -o jsonpath='{.status}' | jq"
+----
++
+** To monitor the Velero logs, enter the following command:
++
+[source,terminal]
+----
+$ oc logs -n openshift-adp -ldeploy=velero -f
+----
+
+.Verification
+
+When the `status.phase` field is `Completed`, the backup process is considered complete.
```

Comment:
AFAIK, "verification" is a step list, so this would get a bullet as a single-step proc
```suggestion
* When the `status.phase` field is `Completed`, the backup process is considered complete.
```